presto-docs/src/main/sphinx/release/release-0.210.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Release 0.210
+=============
+
+General Changes
+---------------
+
+* Fix planning failure when aliasing columns of tables containing hidden columns (:issue:`11385`).
+* Fix correctness issue when ``GROUP BY DISTINCT`` terms contain references to the same column
+  using different syntactic forms (:issue:`11120`).
+* Fix failures when querying information_schema tables using capitalized names.
+* Remove min/max intersection heuristics from equality stats calculator.
+* Improve performance when converting between ``ROW`` types.
+* Remove user cpu time tracking.
+* Make cost-based decision when selecting join distribution type.
```

Comment:
This sounds strange. I think this is trying to say
```
* Select join distribution type automatically using estimated cost.
```
However, I thought that we already had this and it was determined by a config property?

Reply:
cc @arhimondr